Add Matomo Tracking to Docs site.

cburgdorf opened the pull request:
### What was wrong?

Currently we do not have a good insight into our target group. In order to improve both py-ethpm itself and the documentation, it makes sense to aggregate some data.

### How was it fixed?

Add Matomo Tracking to Docs site.

Matomo is an Open Source web analytics platform that allows us
to get better insights and optimize for our audience without
the negative consequences of other compareable platforms.

Read more: https://matomo.org/why-matomo/

#### Cute Animal Picture

![put a cute animal picture link inside the parentheses](http://sacramento.cbslocal.com/wp-content/uploads/sites/15909776/2013/04/baby-sea-turtles.jpg)

pipermerriam commented on docs/conf.py:
```diff
@@ -89,6 +89,9 @@
 html_static_path = ['_static']
 
+def setup(app):
+    app.add_js_file("js/matomo.js")
```
Looks like maybe you forgot to add the actual file?

cburgdorf replied:
Uh, yeah, and I also seem to have added a bunch of unrelated changes. Looks screwed up. Gonna fix it in a minute.

cburgdorf replied:
Ok, PR should be fixed.

cburgdorf committed:
Add Matomo Tracking to Docs site.

Matomo is an Open Source web analytics platform that allows us
to get better insights and optimize for our audience without
the negative consequences of other compareable platforms.

Read more: https://matomo.org/why-matomo/

njgheorghita commented:
@cburgdorf This repo is considered "deprecated" (though, this reminds me I should go through and mark so in the readme) in favor of the `ethpm` module in `web3`. The docs were also moved to the [web3](https://web3py.readthedocs.io/en/stable/ethpm.html) repo. I'm not sure if it's worthwhile to gather data from this repo? But if you feel otherwise, I'm happy to merge this in.

cburgdorf commented:
Oh, good to know! I added https://py-ethpm.readthedocs.io to the Matomo Tracker but if that is being served directly from the web3 docs in the future than there's nothing left to do here. Closing.

njgheorghita commented:
@cburgdorf FYI - https://py-ethpm.readthedocs.io/ is being served from this repo - and is also being deprecated for the `ethpm` docs in `web3` [here](https://web3py.readthedocs.io/en/stable/ethpm.html), so it's probably worth removing the old docs link from matomo.

cburgdorf commented:
No problem. I pinged devops to remove it from the dashboard again (I can't do that myself)
